### PR TITLE
NXP-26348: use blob digest as file version

### PR DESCRIPTION
**Pull request by troger**

### CODE REVIEW

File: nuxeo-wopi/src/main/java/org/nuxeo/wopi/jaxrs/FilesEndpoint.java [hunk 1]
```diff
@@ -227,7 +228,9 @@ public Object getFile(@HeaderParam(MAX_EXPECTED_SIZE) String maxExpectedSizeHead
             throw new PreConditionFailedException();
         }
 
-        logResponse(OPERATION_GET_FILE, OK.getStatusCode());
+        String itemVersion = blob.getDigest();
+        response.addHeader(ITEM_VERSION, itemVersion);
+        logResponse(OPERATION_GET_FILE, OK.getStatusCode(), ITEM_VERSION, itemVersion);
```
Comment (ataillefer) [inline nuxeo-wopi/src/main/java/org/nuxeo/wopi/jaxrs/FilesEndpoint.java]: Can't use `buildItemVersionResponse` here?

File: nuxeo-wopi/src/main/java/org/nuxeo/wopi/jaxrs/FilesEndpoint.java [hunk 2]
```diff
@@ -393,7 +401,9 @@ protected Object unlockOrRefresh(String operation, String lock, boolean unlock)
                         return privilegedSession.removeLock(doc.getRef());
                     });
                 }
-                logResponse(operation, OK.getStatusCode());
+                String itemVersion = blob.getDigest();
+                response.addHeader(ITEM_VERSION, itemVersion);
+                logResponse(operation, OK.getStatusCode(), ITEM_VERSION, itemVersion);
```
Comment (ataillefer) [inline nuxeo-wopi/src/main/java/org/nuxeo/wopi/jaxrs/FilesEndpoint.java]: `buildItemVersionResponse`?

**Comment by kevinleturc** [comment]
👍

**Comment by ataillefer** [comment]
:+1:

File: nuxeo-wopi/src/main/java/org/nuxeo/wopi/jaxrs/FilesEndpoint.java (hunk 1 above)
Comment (troger) [reply nuxeo-wopi/src/main/java/org/nuxeo/wopi/jaxrs/FilesEndpoint.java]: Nop, we need to return the `blob` here, not am empty 200 response.

Comment (troger) [reply nuxeo-wopi/src/main/java/org/nuxeo/wopi/jaxrs/FilesEndpoint.java]: I will use `buildItemVersionResponse` and ignore its return value.

**Pull request merged**